src/components/Whiteboard/Element/ConnectedElement.tsx

```diff
@@ -57,6 +60,14 @@ export const ConnectedElement = ({
       } else if (element.kind === 'triangle') {
         return <TriangleDisplay {...element} {...otherProps} />;
       }
+    } else if (element.type === 'image') {
+      return (
+        <ImageDisplay
+          baseUrl={widgetApi.widgetParameters.baseUrl ?? ''}
```

Comment:
`baseUrl` should be available if widget is configured, right? It would be better to thow an Error not to assume any other state that is not possible.

Reply:
I've added a console log here and return `null` if the base URL is unknown.
There is an issue in our backlog to provide user feedback if there are image loading errors.